client/me/purchases/manage-purchase/notices.jsx

```diff
@@ -921,6 +922,20 @@ class PurchaseNotice extends Component {
 		);
 	}
 
+	renderNonProductOwnerNotice() {
+		const { translate } = this.props;
+
+		return (
+			<Notice
+				showDismiss={ false }
+				status="is-info"
+				text={ translate(
+					'This product was purchased by a different WordPress.com account. To manage this product, log in to that account or contact the account owner.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'This plan was purchased by a different WordPress.com account. To manage this plan, log in to that account or contact the account owner.' )` *ES Score: 7*

Reply:
This is a more generic use of the same sentence. We'll need to update it.